index.js

```diff
@@ -117,6 +117,8 @@ function generateLayoutInternal(options, callback) {
         }
         mapnik.Image.fromSVGBytes(img.svg, mapnikOpts, function(err, image) {
             if (err && err.message.match(/image created from svg must be \d+ pixels or fewer on each side/) && options.removeOversizedIcons) return callback(null, null);
+            // Produce a null result if no width or height attributes. The error message from mapnik has a typo "then"; account for potential future fix to "than".
+            if (err && err.message.match(/image created from svg must have a width and height greater (then|than) zero/)) return callback(null, null);
```

Comment:
wow well done anticipating a fix for `then|than` 💀 – this typo has been haunting mapnik users forever.